pycroft/helpers/printing/__init__.py

```diff
@@ -270,6 +270,181 @@ def generate_user_sheet(user, user_id, plain_password, generation_purpose=''):
     return buf.getvalue()
 
 
+def generate_wifi_user_sheet(user, user_id, plain_password):
+    """Create a „wifi“ datasheet for the given user
+
+    :param User user: A pycroft user
+    :param str user_id: The user's ID.  It has to be given extra,
+        because the user_id is not appearent given the ORM object
+        itself; encoding is done in the library.
+    :param str plain_password: The password
+    """
+    # Anlegen des PDF Dokuments, Seitengröße DIN A4 Hochformat)
+    buf = BytesIO()
+    pdf = SimpleDocTemplate(buf, pagesize=A4,
+                            rightMargin=2 * cm,
+                            leftMargin=2 * cm,
+                            topMargin=0.5 * cm,
+                            bottomMargin=0.5 * cm)
+    style = getStyleSheet()
+    story = []
+
+    PAGE_WIDTH = defaultPageSize[0]
+    PAGE_HEIGHT = defaultPageSize[1]
+
+    # HEADER
+    im_web = Image(ASSETS_WEB_FILENAME, 0.4 * cm, 0.4 * cm)
+    im_house = Image(ASSETS_HOUSE_FILENAME, 0.4 * cm, 0.4 * cm)
+    im_email = Image(ASSETS_EMAIL_FILENAME, 0.4 * cm, 0.4 * cm)
+    im_fb = Image(ASSETS_FACEBOOK_FILENAME, 0.4 * cm, 0.4 * cm)
+    im_t = Image(ASSETS_TWITTER_FILENAME, 0.4 * cm, 0.4 * cm)
+    im_logo = Image(ASSETS_LOGO_FILENAME, 3.472 * cm, 1 * cm)
+
+    if user.room:
+        shortinfo = Paragraph('{dorm}<br/>{name}<br/>{level}/{room}'.format(
+            dorm=str(user.room.building.short_name),
+            name=user.name,
+            level=str(user.room.level),
+            room=str(user.room.number)
```

Comment:
The `str()`s are unnecessary, because `format()`/[`__format__`](https://docs.python.org/3/reference/datamodel.html#object.__format__) does that for you.